src/config/config.cc

```diff
@@ -478,6 +478,12 @@ void Config::initFieldCallback() {
          if (cluster_enabled) srv->slot_migrator->SetSequenceGapLimit(sequence_gap);
          return Status::OK();
        }},
+      {"log-level",
+       [this](Server *srv, const std::string &k, const std::string &v) -> Status {
+         if (!srv) return Status::OK();
+         FLAGS_minloglevel = log_level;
```

Comment:
just a nit, should we not change it when `FLAGS_minloglevel` and `log_level` is equal?

Reply:
i am not very familiar with this (the details), if it's the same, is there any side effect of changing it?

Reply:
Not so important, it will write a global variable, causing some cache invalid. But it's not important